<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rasterer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="MainTab">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="RastererTab">
       <attribute name="title">
        <string>Rasterer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>1095</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Image settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <property name="leftMargin">
               <number>20</number>
              </property>
              <property name="rightMargin">
               <number>20</number>
              </property>
              <item>
               <widget class="QWidget" name="widget" native="true">
                <property name="maximumSize">
                 <size>
                  <width>180</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMaximumSize</enum>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>20</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QPushButton" name="pushButtonChooseImage">
                  <property name="text">
                   <string>Choose image</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Crop settings:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxLockLR">
                  <property name="text">
                   <string>Lock left/right</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="combLineEditLabelCropLeft">
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="cropLeftLabel">
                    <property name="maximumSize">
                     <size>
                      <width>52</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Crop Left</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxCropLeft">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cropLeftType">
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>px</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>%</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="combLineEditLabelCropRight">
                  <item>
                   <widget class="QLabel" name="cropRightLabel">
                    <property name="maximumSize">
                     <size>
                      <width>52</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Crop Right</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxCropRight">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cropRightType">
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>px</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>%</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBoxLockTB">
                  <property name="text">
                   <string>Lock top/bottom</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="combLineEditLabelCropTop">
                  <item>
                   <widget class="QLabel" name="cropTopLabel">
                    <property name="maximumSize">
                     <size>
                      <width>52</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Crop Top</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxCropTop">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cropTopType">
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>px</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>%</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="combLineEditLabelCropBottom">
                  <item>
                   <widget class="QLabel" name="cropBottomLabel">
                    <property name="maximumSize">
                     <size>
                      <width>52</width>
                      <height>20</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Crop Bottom</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxCropBottom">
                    <property name="maximumSize">
                     <size>
                      <width>80</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="maximum">
                     <number>999999</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cropBottomType">
                    <property name="maximumSize">
                     <size>
                      <width>40</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>px</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>%</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGraphicsView" name="graphicsViewFullImage">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="backgroundBrush">
                 <brush brushstyle="NoBrush">
                  <color alpha="255">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>PDF settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <item>
                 <widget class="QLabel" name="Rast_label_1">
                  <property name="text">
                   <string>Rasterizer settings:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Desired size (mm)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxDesiredSize">
                    <property name="maximum">
                     <number>99999999</number>
                    </property>
                    <property name="value">
                     <number>401</number>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBoxDesiredAxis">
                    <property name="maximumSize">
                     <size>
                      <width>50</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <item>
                     <property name="text">
                      <string>heigh</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>wide</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Margin x (mm)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxPDFMarginX">
                    <property name="value">
                     <number>5</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Margin y (mm)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBoxPDFMarginY">
                    <property name="value">
                     <number>5</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBoxMarking">
                    <item>
                     <property name="text">
                      <string>Half cross</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Full cross</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Full line</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>No marking</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Page layout</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBoxPageRotation">
                    <item>
                     <property name="text">
                      <string>vertical</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>horizontal</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QPushButton" name="pushButtonCreatePDF">
                  <property name="text">
                   <string>Create PDF</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QGraphicsView" name="graphicsViewPDFImage">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="backgroundBrush">
                 <brush brushstyle="NoBrush">
                  <color alpha="0">
                   <red>0</red>
                   <green>0</green>
                   <blue>0</blue>
                  </color>
                 </brush>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>SplitPDF</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>1091</width>
          <height>471</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <item row="0" column="0">
          <widget class="QGraphicsView" name="graphicsView"/>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
